lighthouse-cli/test/smokehouse/test-definitions/tricky-metrics/expectations.js

```diff
@@ -7,7 +7,9 @@
 
 /**
  * @type {Array<Smokehouse.ExpectedRunnerResult>}
- * Expected Lighthouse audit values for tricky metrics tests
+ * Expected Lighthouse audit values for tricky metrics tests that previously failed to be computed.
+ * We only place lower bounds because we are checking that these metrics *can* be computed at that
+ * we wait long enough to compute them. Upper bounds aren't very helpful here and tend to cause flaky failures.
```

Comment:
I can accept this reasoning but then it seems like the `should be around 10s` etc comments below should be more like `should be at least 10s` :P

Reply:
sure thing